util/missingcrypto.txt

```diff
@@ -738,6 +738,7 @@ IPAddressChoice_it(3)
 IPAddressFamily_it(3)
 IPAddressOrRange_it(3)
 IPAddressRange_it(3)
+ISSUER_SIGN_TOOL_it(3)
```

Comment:
Please just add it to the X509_dup page as well. There are already other _it objects there. I've noticed there are some typos in description of the _it objects there so if you want (not required), you can fix them as well.

Reply:
fix commited

Reply:
BTW, @NMorozxov congrats.  You took some feedback on your PR and went back and made other manpages comply.  Nice work!